src/main/cup/parser.cup

```diff
@@ -52,8 +52,12 @@ nonterminal      ConditionalExpression, AssignmentExpression, Expression;
 nonterminal      PackageDelcaration_opt, PackageDelcaration, ImportDeclaration_opt, ImportDeclaration;
 nonterminal      TypeParameters_opt, TypeParameters, Superclass_opt, Superclass, Superinterfaces_opt, Superinterfaces;
 nonterminal      Throws_opt, Throws, VariableModifier_opt, VariableModifier, TypeArguments_opt, TypeArguments;
+<<<<<<< HEAD
+nonterminal      Assignment,ReferenceType;
+=======
 nonterminal      PreIncrementExpression, PreDecrementExpression;
 nonterminal      ExpressionName, PostIncrementExpression, PostDecrementExpression, ArrayCreationExpression;
+>>>>>>> vthai/master
```

Comment:
These <<<<< HEAD  >>>> vthai/master indicates there is a git merge conflict... Are you sure that it pass build :)